src/tests/e2e/landing.test.e2e.js

```diff
@@ -6,7 +6,8 @@ import { sleep } from './testUtils'
 const timeout = 180000
 
 describe('Landing page tests', () => {
-  beforeAll(async () => {
+  beforeEach(async () => {
+    await jestPuppeteer.resetBrowser()
```

Comment:
we shouldn't need to reset browser among each test in landing.

Reply:
yes, this will be another issue